cookbook/deployment/cluster/README.rst

```diff
@@ -132,12 +132,3 @@ Using remote Flyte gives you the ability to:
 - Use caching to avoid calling the same task with the same inputs (for the same version)
 - Portability: You can reference pre-registered entities under any domain or project within your workflow code
 - Sharable executions: you can easily share links to your executions with your teammates
-
-.. toctree::
-   :caption: Cluster Deployment
-   :hidden:
-
-   config_flyte_deploy
-   auth_setup
-   monitoring
-   access_cloud_resources
```

Comment:
Don't remove these lines. It's required to render all the concerned RST files.

Reply:
see #245 to see why we shouldn't add an explicit toctree in sphinx gallery `README.rst` files.